src/main/java/appeng/spatial/StorageCellLot.java

```diff
@@ -0,0 +1,85 @@
+package appeng.spatial;
+
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * A lot inside the storage cell world that is assigned to a specific storage cell.
+ */
+public class StorageCellLot {
+
+    private static final int MAX_SIZE = 512;
+
+    /**
+     * Id of the lot.
+     */
+    private final int id;
+
+    /**
+     * The storage size of this dimension. This is dicateted by the pylon structure
+     * size used to perform the first transfer into this dimension. Once it's set,
+     * it cannot be changed anymore.
+     */
+    private final BlockPos size;
```

Comment:
Is this absolute or relative to the origin? Assuming that uses a `BlockPos` to encode a `Vector3i`-like point.